packages/@aws-cdk/aws-elasticloadbalancing/test/loadbalancer.test.ts

```diff
@@ -89,6 +89,78 @@ describe('tests', () => {
     });
   });
 
+  test('add an Instance as load balancing target', () => {
+    // GIVEN
+    const stack = new Stack();
+    const vpc = new Vpc(stack, 'VCP');
+    const securityGroup = new SecurityGroup(stack, 'simple-instance-1-sg',
+      {
+        vpc,
+        allowAllOutbound: true, // will let your instance send outboud traffic
+        securityGroupName: 'simple-instance-1-sg',
+      },
+    );
+
+    // lets use the security group to allow inbound traffic on specific ports
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(22),
+      'Allows SSH access from Internet',
+    );
+
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(80),
+      'Allows HTTP access from Internet',
+    );
+
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(443),
+      'Allows HTTPS access from Internet',
+    );
+    const instance = new Instance(stack, 'targetInstance', {
+      vpc: vpc,
+      instanceType: InstanceType.of( // t2.micro has free tier usage in aws
+        InstanceClass.T2,
+        InstanceSize.MICRO,
+      ),
+      machineImage: MachineImage.latestAmazonLinux({
+        generation: AmazonLinuxGeneration.AMAZON_LINUX_2,
+      }),
+      securityGroup,
+    });
+    const connections = new Connections({
+      peer: Peer.ipv4('666.666.666.666/666'),
+    });
+    const elb = new LoadBalancer(stack, 'LB', {
+      vpc,
+      healthCheck: {
+        interval: Duration.minutes(1),
+        path: '/ping',
+        protocol: LoadBalancingProtocol.HTTPS,
+        port: 443,
+      },
+    });
+
+    // WHEN
+    elb.addListener({ externalPort: 80, internalPort: 8080 });
+    elb.addTarget(new InstanceTarget(instance, connections));
```

Comment:
OK I see that `attachToClassicLB` is tested via the `addTarget` API since `elb.addTarget` is implemented like this:

```ts
  public addTarget(target: ILoadBalancerTarget) {
    target.attachToClassicLB(this);

    this.newTarget(target);
  }
```

TO me, it seems like `attachToClassicLB` should be the place where we implement the 'attaching' code and it's quite odd that it's empty...